main.c

```diff
@@ -71,10 +74,60 @@ static struct kanshi_profile *match(struct kanshi_state *state,
 }
 
 
+static void exec_command(char *cmd) {
+	pid_t pid, child;
+	// Fork process
```

Comment:
This comment could be removed, it doesn't provide more information than the line below